### PR TITLE
[POSG78] Ajustes inscricao

### PR DESCRIPTION
**Pull request by mblucs**
https://mate85-equipe03.atlassian.net/jira/software/projects/POSG/boards/3?selectedIssue=POSG-78

### CODE REVIEW

File: src/Components/BtnSubmitLoading.tsx [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+      form={formId}
+      size="large"
+      disabled={loading}
+    >
+      {loading ? <Loading /> : label}
+    </Button>
   );
 }
+
+BtnSubmitLoading.defaultProps = {
```
Comment (BeatrizCerqueira) [inline src/Components/BtnSubmitLoading.tsx:30]: Legal isso, n conhecia

File: src/pages/Cadastro/Cadastro.tsx [hunk 2]
```diff
@@ -267,12 +268,16 @@ export default function Cadastro() {
             </FormControl>
           </form>
         </CardContent>
-
-        <BtnSubmitLoading
-          label="Enviar"
-          formId="sign-up-form"
-          loading={loading}
-        />
+        <CardActions sx={{ px: { xs: 5, sm: 10 } }}>
```
Comment (BeatrizCerqueira) [inline src/pages/Cadastro/Cadastro.tsx:271]: 🙏

File: src/pages/Edital/Inscricao/Components/AttachedFile.tsx [hunk 3]
```diff
@@ -43,38 +43,68 @@ export default function AttachedFile({
   };
 
   return (
-    <ListItem
-      sx={{ my: 1, py: 1, cursor: "pointer", border: 1, borderRadius: 1 }}
+    <ListItemButton
```
Comment (BeatrizCerqueira) [inline src/pages/Edital/Inscricao/Components/AttachedFile.tsx:46]: Boaaa, n tinha pensado nisso

File: src/pages/Edital/Inscricao/View.tsx [hunk 4]
```diff
@@ -241,32 +220,34 @@ export default function Inscricao() {
...
-                mt: 3,
+                m: 3,
               }}
             >
-              <Typography variant="body1" sx={{ pb: 1 }}>
-                Marque as opções que se aplicam *
-              </Typography>
-
+              <FormLabel component="legend">
+                Marque as opções que se aplicam
```
Comment (BeatrizCerqueira) [inline src/pages/Edital/Inscricao/View.tsx:227]: Kd o asterisco

**Review by BeatrizCerqueira — APPROVED** [review]

File: src/pages/Edital/Inscricao/View.tsx (hunk 4 above)
Comment (mblucs) [reply src/pages/Edital/Inscricao/View.tsx:227]: aparece pq o campo ta required

**Pull request merged**